CHANGELOG.md

```diff
@@ -3,6 +3,24 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.40.1] - 2025-02-24
+
+### Releasing
+
+#### Features
+
+- Introduce release script (#446) ([2e252e2](https://github.com/vectordotdev/helm-charts/commit/2e252e20bfcb560847481b02e7d484f6302b48ad))
```

Comment:
I think this is a chore, too. We tend to keep the changelog focused on user-facing features.